pkgs/by-name/lx/lx-music-sync-server/package.nix

```diff
@@ -21,14 +22,25 @@ buildNpmPackage rec {
 
   buildInputs = [
     nodejs
+    bash
   ];
 
   postInstall = ''
     cp -r server $out/lib/node_modules/lx-music-sync-server
     mkdir -p $out/bin
```

Comment:
Nit: I'd prefer `install -D server "$out/lib/node_modules/lx-music-sync-server"` and `install -d "$out/bin"`.